CondCore/Utilities/o2o/deployed/offline/setupO2O.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+
+# deployment-specific params
+BASEDIR=/data/O2O
+RELEASE=CMSSW_10_0_5
```

Comment:
@ggovi and here...

Reply:
these are examples of the scripts created by deploy_online, currently running at P5